src/js/base/js-numbers.js

```diff
@@ -1080,7 +1080,7 @@ define("pyret-base/js/js-numbers", function() {
       if (typeof(n) === 'number') {
         n = makeBignum(n);
       }
-      return onBignums(m, n);
+      return onBignums(m, n, errbacks);
```

Comment:
Adding this is good, but then make sure that every use of `makeIntegerBinop` has 3-argument functions, even if they don't use `errbacks`, so that we don't have multi-arity functions